code/global.dm

```diff
@@ -185,3 +185,17 @@ var/static/icon/ingame_hud_med = icon('icons/mob/hud_med.dmi')
 //Keyed list for caching icons so you don't need to make them for records, IDs, etc all separately.
 //Could be useful for AI impersonation or something at some point?
 var/static/list/cached_character_icons = list()
+
+var/cans_opened_temp = 0
```

Comment:
I feel like these are the sort of thing that's supposed to live on GLOB

Reply:
I have no idea what i am doing, really.